docs/architecture/deployment/emr.md

```diff
@@ -22,7 +22,7 @@ Livy does not work with Python version 3.8 and above.
 
 Navigate to Amazon EMR service and create a cluster. Under **Application bundle** select **Custom**.
 
-Choose appropriate applications to include in your installation. At a minimum, please make sure sure **Livy** , **Spark** and and **Hive** are included in the install.
+Choose appropriate applications to include in your installation. At a minimum, please make sure sure **Livy** and **Spark** are included in the install.
```

Comment:
Overall comment: 
The content of this page seems duplicated with the content on this page: 
http://localhost:3000/low-code-spark/fabrics/EMR-fabric-configuration

Line 38: 
Do we require all https ports, or 443 or 8443?
https://www.hostinger.com/tutorials/https-port

Line 61: 
Nitpick: this screenshot is slightly cutting off the text.